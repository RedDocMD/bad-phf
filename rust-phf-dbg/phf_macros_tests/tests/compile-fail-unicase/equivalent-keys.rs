use unicase::UniCase;
use phf::phf_map;

static MAP: phf::Map<UniCase<&'static str>, isize> = phf_map!(
    UniCase::ascii("FOO") => 42,
    UniCase::ascii("foo") => 42, //~ ERROR duplicate key UniCase("FOO")
);

fn main() {}
