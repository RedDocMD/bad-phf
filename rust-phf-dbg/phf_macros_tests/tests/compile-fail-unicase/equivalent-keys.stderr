error: duplicate key
 --> tests/compile-fail-unicase/equivalent-keys.rs:6:5
  |
6 |     UniCase::ascii("foo") => 42, //~ ERROR duplicate key UniCase("FOO")
  |     ^^^^^^^^^^^^^^^^^^^^^
